CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Renamed the protobuf file of the transaction prover to `tx_prover.proto` (#1110).
 - [BREAKING] Renamed `AccountData` to `AccountFile` (#1116).
 - Implement transaction batch prover in Rust (#1112).
+- Added environment variables to the `miden-proving-service` (#1120).
```

Comment:
I would change this to: "Refactored config file for `miden-proving-service` to be based on environment variables."